src/components/Card/WorkoutCard.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+function WorkoutCard(props) {
+  return (
+    <div class="max-w-sm rounded overflow-hidden shadow-lg font-poppins ">
+      <img
+        class="w-full"
+        src="https://cdn.muscleandstrength.com/sites/default/files/field/feature-wide-image/workout/12-week-womens-bikini-prep-workout-1_0.jpg"
+        alt="Sunset in the mountains"
+      />
+      <div class="px-6 py-4">
+        <div class="flex justify-center items-center font-bold text-xl  text-center m-10 text-darkgray">
+          FAT LOSE WORKOUTS
+        </div>
+      </div>
+      <div class="flex justify-around m-5">
+        <div class="">
+          <svg
```

Comment:
`svg` should be in a sperate file in the same directory and imported and used here